### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by juliusgeo**

### CODE REVIEW

**Review by blink1073 — APPROVED** [review]
Thanks!

File: changelog.rst [hunk 1]
```diff
@@ -2,11 +2,22 @@
 Changelog
 =========
 
+Changes in version 1.2.0
+------------------------
+- Added ability to explain command options
```
Comment (ShaneHarvey) [inline changelog.rst:7]: Can you rephrase this? Best would be to mention the feature or new parameter name itself so that users can find it.

**Review by ShaneHarvey — CHANGES_REQUESTED** [review]

**Review by ShaneHarvey — APPROVED** [review]
